core/components/molecules/emptyState/EmptyStateActions.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classNames from 'classnames';
+import { BaseProps } from '@/utils/types';
+import EmptyStateContext from './EmptyStateContext';
+
+export interface EmptyActionProps extends BaseProps {
+  /**
+   * The content to be displayed within the component
+   * Can be any renderable React Node.
+   */
+  children: React.ReactNode;
+}
+
+const EmptyStateActions = (props: EmptyActionProps) => {
+  const { children, className, ...rest } = props;
+  const contextProp = React.useContext(EmptyStateContext);
+
+  const { size = 'standard' } = contextProp;
+
+  const actionWrapperClasses = classNames(
+    {
+      [`EmptyState-actions--${size}`]: true,
+    },
+    'd-flex',
+    className
+  );
+
+  return (
+    <div {...rest} data-test="DesignSystem-EmptyState--Actions" className={actionWrapperClasses}>
```

Comment:
`EmptyStateActions` accept `data-test` prop but it is not correctly applied on this and will always be override by the default value `data-test="DesignSystem-EmptyState--Actions"`. Please check this once and also add a test case for this.

Reply:
done